### DISCUSSION TITLE
[Shipped] Elevate Page Performance with the New Web Vitals Workflow

### DISCUSSION THREAD

**Discussion by zoesyc** [getsentry/sentry#59620] [Launch Feedback]
We understand that fast-loading, seamless websites are the key to user satisfaction, search engine success, and revenue growth. With this in mind, Web Vitals are your essential metrics for optimizing users’ experience. 

Introducing Sentry’s new Web Vitals workflow, designed to help you streamline your efforts and achieve peak page performance! 

Starting today, you can find the **Web Vitals** feature tab right under **Performance** in the sidebar. 

![Screenshot 2023-11-06 at 12 44 08 PM](https://github.com/getsentry/sentry/assets/38118634/9f25fee8-281e-4ada-b90d-57ab29cd413a)

<h2>New concepts</h2>

- **<h4>[Performance score](https://docs.sentry.io/product/performance/web-vitals/#performance-score)</h4>** It can be difficult to compare performance across 5 different metrics with varying standards. So we've calculated a single performance score using normalized web vitals ([similar to Lighthouse / CrUX](https://www.desmos.com/calculator/o98tbeyt1t)) so you can easily compare the performance across multiple pages and just as easily understand how your app is performing as a whole.  

- **<h4>[Opportunity](https://docs.sentry.io/product/performance/web-vitals/#opportunity)</h4>** The advantage of having a single performance score is that Sentry can now show you which pages have the highest potential for improving your site's overall performance. 'Opportunity' has 2 main factors: performance and traffic. For example, a page that scores 20 with 1000 page loads per day will have a higher potential (opportunity) than a page that scores 70 with 30 page loads per day.

<h2> We want to hear from you!</h2>
We're rethinking our troubleshooting workflows and want to know if this is working for you! We'd like to better understand:
<br></br>

1. Are the new concepts (performance score, opportunity) helping you better diagnose your site's performance? Are they intuitive or useful in helping you get down to root causes?
2. What kind of problems motivate you to monitor or troubleshoot your web vitals performance? 
3. What are your usual next steps when you see that, for example, LCP is bad for a given page?
4. What are some additional features you'd like to see?

To be able to use Web Vitals, you'll need to have Performance set up - here's our [setup guide](https://docs.sentry.io/product/performance/getting-started/) for instructions. You can learn more about this update in our [docs](https://docs.sentry.io/product/performance/web-vitals/). 

Feel free to share your thoughts in the discussion below - or you can find us on [Discord](https://discord.com/invite/Ww9hbqr) and on [X](https://twitter.com/getsentry).

**Comment by srtab** [comment]
That's great! Is it possible to enable this feature on self-hosted installations? If so, how? Is any documentation available? I would love to test ir. Thank you!
